doc/flume_extensions_catalogue/orion_sth_sink.md

```diff
@@ -248,24 +253,33 @@ In case of using hashes as part of the collection names and to let the user or d
 
 [Top](#top)
 
+###<a name="section2.3.2"></a>About batching
+Despite `OrionSTHSink` allows for batching configuration, it is not true it works with real batches as the rest of sinks. The batching mechanism was designed to accumulate NGSI-like notified data following the configured data model (i.e. by service, service path, entity or attribute) and then perform a single bulk-like insert operation comprising all the accumulated data.
+
+Nevertheless, STH storage aggregates data through updates, i.e. there are no inserts but updates of certain pre-populated collections. Then, these updates implement at MongoDB level the expected aggregations of STH (sum, sum2, max and min).
+
+The problem with such an approach (updates versus inserts) is there is no operation in the Mongo API enabling the update of a batch. As much, there exists a `updateMany` operation, but it is about updating many collections with a single data (the updated collections are those matching the given query).
+
+Thus, `OrionSTHSink` does not implement a real batching mechanism as usual. Please observe the batching accumulation is still valid, since many events may be accumulated and processed at the same time, even in the case of configuring a batch size of 1, a single notification may include several context elements. The difference with regard to the other sinks is the events within the batch will be processed one by one after all.
+
+[Top](#top)
+
 ##<a name="section4"></a>Programmers guide
 ###<a name="section4.1"></a>`OrionSTHSink` class
-`OrionSTHSink` extends `OrionMongoBaseSink`, which as any other NGSI-like sink extends the base `OrionSink`. The methods that are extended are by `OrionMongoBaseSink` are:
+`OrionSTHSink` extends `OrionMongoBaseSink`, which s any other NGSI-like sink, extends the base `OrionSink`. The methods that are extended are:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/868f39a587310f3ac46e88aa439fbdf01f9a6a90